rfcs/xxxx-standard-gh-labels.md

```diff
@@ -0,0 +1,118 @@
+---
+created-on: 2021-03-23   (date the RFC was created)
+freshness: YYYY-MM-DD    (date the RFC was last reviewed for freshness)
+owner: @CalvinR (owner responsible for freshness review)
+status: draft
+tags:
+  - github
+  - project_management
+---
+
+[La version française suit.](#résumé)
+
+# CDS Github Label Standard
+
+## Summary
+Create an organization wide standard for Github labels to ensure a consistent way of identifying issues across the org as well as allow customization of labels to happen at the business unit level.
+
+## Motivation
+
+Currently every github repo has to modify existing Github labels upon creation of a new repo or use the default labels. Unfortunately GH does not provide bilingual labels out of the box and so we are non-compliant with OLA (Official Languages Act) requirements. If we customize lables for a business unit (BU) we will then have to manually copy labels into every repo that is owned by that BU.
+
+## Proposal
+A standard baseline of labels to be applied across the cds-snc github organization. This standard will be enforced through a github action on each repo, and will allow for a consistent way to identify issues across the organization and will allow for products, business units or specific products to extend this baseline with labels that fit their needs. This will also allow for CDS to ensure that we are respecting our requirements to communicate with people in their official language of choice.
+
+See https://github.com/cds-snc/covid-alert-server-labels for an example of what is being proposed to implement this solution. 
+
+The source of truth for these labels will reside in github and what is put forth in this document is just to start the discussion. The understanding is that if this Standard is accepted that these labels will be copied into github and maintenance and discussion will continue in those repositories. 
+
+### Baseline Labels
+This list of labels will be standardized across the organization and will be mandatory, they will apply high level groupings and statuses and will allow for a consistent way of talking about issues across projects 
+
+This consistent vocabulary will enable us to create reporting tools that can be applied to any project using the system and will provide a consistent view into the state of all CDS products. It should also reduce cognitive burden when onboarding to the project. 
+
+The list of proposed baseline labels can be seen in Appendix A
+
+### Product Specific Labels
+
+Each product will be allowed to apply their own unique set of labels alongside the baseline labels. 
+
+
+- ```
+  Name: High Priority | Haute priorité
+  Description: 
+  Colour: 
+- ```
+  Name: Medium Priority  | Priorité moyenne
+  Description:
+  Colour:
+- ```
+  Name: Low Priority | Faible priorité
+  Description:
+  Colour
+- ```
+  Name: Bug | Bogue
+  Description:
+  Colour:
+- ```
+  Name: Documentation
+  Description:
+  Colour:
+- ```
+  Name: Security | Sécurité
```

Comment:
I don't think we should propose this. We probably need security issues to _not_ go through the public trackers as they become public but to follow a separate security procedure. This may make it confusing to folks who think will then think security issues should get filed through these trackers.

Reply:
That makes sense but at the same time we have shared some security issues in our repos but mainly before things have made it to production. 

https://github.com/cds-snc/covid-alert-server/issues?q=is%3Aissue+label%3ACCCS+is%3Aclosed+label%3Asecurity

So I think there might be a need for this but I'm open to more discussion around it.

Reply:
Maybe we just need to set the description to something like 'Non-production security issues' ? Then we have the SECURITY.md files which point to where to send production security issues.